.github/workflows/release-automation.yaml

```diff
@@ -218,4 +219,4 @@ jobs:
           delete-branch: true
           signoff: true
           title: ${{ needs.release.outputs.pr_title }}
-          token: ${{ secrets.HEDERA_BOT_TOKEN }}
```

Comment:
There are 2 locations for `HEDERA_BOT_TOKEN` in this file that have been removed. Can we get rid of the `HEDERA_BOT_TOKEN` all together?

Reply:
Roger mentioned this never existed in the first place. Marking resolved.